_rules/role-has-required-states-and-properties-4e8ab6.md

```diff
@@ -28,7 +28,7 @@ For each test target, the [WAI-ARIA required states and properties](https://www.
 
 ## Assumptions
 
-- The applicability of this rule is limited to [explicit semantic roles][] based on an assumption that all native HTML and SVG elements have native attributes that are mapped to all of the [required states and properties](https://www.w3.org/TR/wai-aria/#requiredState) for the [implicit semantic role][] of the element.
+- The applicability of this rule is limited to [explicit semantic roles](#explicit-role) based on an assumption that all native HTML and SVG elements have native attributes that are mapped to all of the [required states and properties](https://www.w3.org/TR/wai-aria/#requiredState) for the [implicit semantic role][] of the element.
```

Comment:
The link is not missing. It is using the "short link" syntax of markdown. The actual link is at the very bottom of the file (visible only in `raw view` since the whole point is that markdown replaces it).

Using the short syntax allow to use the same link several times without copying the full destination every single time.

Reply:
For some reason the link isn't working on this instance, although it is working correctly on all the other links on the page implemented the same way.

Reply:
Ah, yes 😞 Was the extra 's'.